### DISCUSSION TITLE
Maximum Applications That Can Be Created in Argo CD

### DISCUSSION THREAD

**Discussion by y509412** [argoproj/argo-cd#5769] [Q&A]
Currently we have an HA mode setup of Argo CD of version 1.5.8. We will have to deploy more than 1000 apps at least, via Argo CD.  What is the maximum number of applications that can be created in Argo CD? Should we tweak any configs if we have to create more than 1000 apps?

**Comment by rumstead** [comment]
I would first suggest upgrading, v2.0.0 was just released :). 

The docs here provide a lot of good information on how to scale up and handle load on ArgoCD, https://argoproj.github.io/argo-cd/operator-manual/high_availability/#scaling-up.
